### PR TITLE
Add Node-Creation in Roads that only have a start- and endpoint

### PR DESCRIPTION
**Pull request by Cathesso**

### CODE REVIEW

File: backend/src/main/java/de/cathesso/scratchamap/controller/MapDataController.java [hunk 1]
```diff
@@ -19,13 +19,13 @@
     public MapDataController(MapDataService mapDataService){this.mapDataService = mapDataService;}
 
     @GetMapping("getNodes")
```
Comment (CultOfTheDeadCrow) [inline backend/src/main/java/de/cathesso/scratchamap/controller/MapDataController.java:21]: not sure if caps in url are necessary

File: backend/src/main/java/de/cathesso/scratchamap/controller/MapDataController.java [hunk 2]
```diff
@@ -19,13 +19,13 @@
...
-    public List<Node> getMarkers(@RequestParam String sWLat, String sWLon, String nELat, String nELon){
-        return mapDataService.getMarkers(sWLat,sWLon,nELat,nELon);
+    public List<Node> getNodes(Principal principal, @RequestParam String sWLat, String sWLon, String nELat, String nELon){
+        return mapDataService.getNodes(principal, sWLat,sWLon,nELat,nELon);
     }
 
     @PostMapping("saveNodes")
```
Comment (CultOfTheDeadCrow) [inline backend/src/main/java/de/cathesso/scratchamap/controller/MapDataController.java:26]: same as above

**Review by CultOfTheDeadCrow — APPROVED** [review]
Very nice

File: backend/src/main/java/de/cathesso/scratchamap/controller/MapDataController.java (hunk 1 above)
Comment (Cathesso) [reply backend/src/main/java/de/cathesso/scratchamap/controller/MapDataController.java:21]: I just wanted idea to stop telling me I made a typo :'D

**Pull request merged**